R/pkg/R/DataFrame.R

```diff
@@ -743,8 +743,12 @@ setMethod("toJSON",
 #' @note write.json since 1.6.0
 setMethod("write.json",
           signature(x = "SparkDataFrame", path = "character"),
-          function(x, path) {
+          function(x, path, mode = "error", ...) {
```

Comment:
Yeap. Default is `SaveMode.ErrorIfExists`[1] which `error` means[2].

[1]https://github.com/apache/spark/blob/50b89d05b7bffc212cc9b9ae6e0bca7cb90b9c77/sql/core/src/main/scala/org/apache/spark/sql/DataFrameWriter.scala#L573
[2]https://github.com/apache/spark/blob/50b89d05b7bffc212cc9b9ae6e0bca7cb90b9c77/sql/core/src/main/scala/org/apache/spark/sql/DataFrameWriter.scala#L70